spec/factories/group_order_article_quantity.rb

```diff
@@ -0,0 +1,9 @@
+require 'factory_bot'
+
+FactoryBot.define do
+
+  # requires order_article
+  factory :group_order_article_quantity do
+  end
```

Comment:
would be good to add `order_article` inside the factory if it's required

Reply:
That was a mistake in my comment - it should have read `# requires group_order_article`.
And actually I intentionally didn't add it to the new factory as it was manually set by my tests anyway. I followed the same pattern, that was used here for example: https://github.com/foodcoops/foodsoft/blob/45a8911ca656253d17ba095e698b232629db0b94/spec/factories/group_order.rb#L5 

But I just added this associations (and all ones this entails) in this PR and all the tests still pass:
https://github.com/foodcoops/foodsoft/pull/810/files
(With that you *could* use `create :group_order_article_quantity` without manually passing a `group_order_article`. I marked the PR as `Draft` as I don't really see the point of this, but if you do feel free to merge it anyway :smile: )

Reply:
Ah, great you followed existing convention! I think it would be useful to add this, great you submitted a PR ... ! :rocket:

Reply:
What I understood of FactoryBot's is that the idea is that each factory can be built by itself. If you want to give some as arguments, fine, but all minimum requirements are best to define in the factory (to avoid boilerplate in tests, unless it is needed because certain test objects need to be related).